remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyDecoder.java

```diff
@@ -45,6 +46,14 @@ public Object decode(ChannelHandlerContext ctx, ByteBuf in) throws Exception {
             }
             return RemotingCommand.decode(frame);
         } catch (Exception e) {
+            in.resetReaderIndex();
+            // user input may be HiRMQ, HiRMQ\n or HiRMQ\r\n
+            if (in.readableBytes() >= 5 || in.readableBytes() <= 7) {
+                String magicCode = in.readCharSequence(5, StandardCharsets.UTF_8).toString();
+                if (magicCode.equalsIgnoreCase("HiRMQ")) {
```

Comment:
I think that it was important to let the user know that the response was sent by `hirmq`, so It needs to be marked on the document

Reply:
Good idea, I have added this feature to document.